selinux_policy_aws/README.md

```diff
@@ -1,69 +1,115 @@
 # SELinux Policy Workshop - AWS Version
 
-![ansible](img/Ansible-Tower-Logotype-Large-RGB-FullGrey-300x124.png)
+![selinux](img/200px-SELinux_logo.svg.png)
 
 `selinux_policy_aws` is a ansible playbook to provision RHEL nodes in AWS, for SELinux workshops. This playbook uses Ansible to wrap Terraform, for provisioning AWS infrastructure and nodes. To find more info about Terraform [check here](https://www.terraform.io/docs/providers/aws/index.html)
 
 These modules all require that you have AWS API keys available to use to provision AWS resources. You also need to have IAM permissions set to allow you to create resources within AWS. There are several methods for setting up you AWS environment on you local machine.
 
-Fill out `env.sh` & Export the AWS API Keys
-
-First, copy env.sh_example to env.sh, and then fill in your API keys.  Once that is complete, source the script, to export your AWS environment variables.
-
-```
-source env.sh
-```
-
 This repo also requires that you have Ansible installed on your local machine. For the most upto date methods of installing Ansible for your operating system [check here](http://docs.ansible.com/ansible/intro_installation.html).
```

Comment:
Why not say, "To view Ansible installation methods according to your operating system"